lib/modules/apostrophe-groups/index.js

```diff
@@ -86,6 +91,37 @@ module.exports = {
       }
     };
 
+    self.enableAddGroupTask = function() {
+      self.apos.tasks.add(self.__meta.name,
+        'add',
+        'Usage: node app apostrophe-groups:add groupname permission1 permission2...\n\nTo list available permissions, run:\n\nnode app apostrophe-permissions:list',
+        function(apos, argv) {
+          var req = self.apos.tasks.getReq();
+          var groupname = argv._[1];
+          if (!groupname) {
+            throw 'You must specify a group name.';
+          }
+          var permissions = argv._.slice(2);
+          return Promise.try(function() {
+            _.each(permissions, function(permission) {
+              if (!_.contains(_.pluck(self.apos.permissions.getChoices(), 'value'), permission)) {
```

Comment:
`_.pluck` removed in favor of `_.map` in lodash 4 - not a problem for us here, but we might want to stop using `_.pluck` in case we ever decide we want to upgrade lodash?